File: app/src/main/java/org/lantern/app/model/BannerAd.java
package org.lantern.app.model;

import com.google.gson.annotations.SerializedName;

public class BannerAd {
    @SerializedName("enabled")
    private boolean enabled;

    @SerializedName("text")
    private String text;

    @SerializedName("url")
    private String url;

    public boolean getEnabled() {
        return enabled;
    }

    public void setEnabled(boolean enabled) {
        this.enabled = enabled;
    }

    public String getText() {
        return text;
    }

    public void setText(final String text) {
        this.text = text;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(final String url) {
        this.url = url;
    }
}

File: android/app/src/main/java/org/getlantern/mobilesdk/model/BannerAd.kt
package org.getlantern.mobilesdk.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class BannerAd {
    @Expose
    @SerializedName("enabled")
    var enabled = false

    @Expose
    @SerializedName("text")
    var text: String? = null

    @Expose
    @SerializedName("url")
    var url: String? = null
}